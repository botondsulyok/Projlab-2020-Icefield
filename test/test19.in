field spawn 1 true 0
entity builder Eskimo spawn
stats spawn
stats builder
ability spawn builder
stats spawn
stats builder