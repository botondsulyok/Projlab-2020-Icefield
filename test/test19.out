Game Over
id: spawn
type: unstable
building: plains0
snow: 0
pickable: null
standers: builder
pullers: builder
masters: builder
neighbours: null
capacity: 1
broken: true

id: builder
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: spawn
type: unstable
building: plains0
snow: 0
pickable: null
standers: builder
pullers: builder
masters: builder
neighbours: null
capacity: 1
broken: true

id: builder
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

